Project.toml

```diff
@@ -17,9 +16,8 @@ TerminalLoggers = "5d786b92-1e48-4d6f-9151-6b4477ca9bed"
 DocStringExtensions = "0.9"
 LinearAlgebra = "1"
 Logging = "1"
-NumericalIntegration = "0.3.3"
 Optim = "1.7"
 QuadGK = "2.9"
 SpecialFunctions = "2.3"
 TerminalLoggers = "0.1"
-julia = "1.9"
+julia = "1.10"
```

Comment:
It's better to keep Julia dependency lower to allow a wider possibility of usage. We should only increment Julia dependency if we use a feature that only exist in higher versions.